jflex/src/main/java/jflex/generator/Emitters.java

```diff
@@ -0,0 +1,66 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * JFlex 1.8.0-SNAPSHOT                                                    *
+ * Copyright (C) 1998-2018  Gerwin Klein <lsf@jflex.de>                    *
+ * All rights reserved.                                                    *
+ *                                                                         *
+ * License: BSD                                                            *
+ *                                                                         *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+package jflex.generator;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+import jflex.core.DFA;
+import jflex.core.LexParse;
+import jflex.core.Options;
+import jflex.core.Out;
+
+/** Factory class for Emitter */
+public final class Emitters {
+
+  private Emitters() {}
+
+  /**
+   * Create Emitter that writes to file
```

Comment:
nit: in the google code style, this would be "Creates ..."
(no action needed, jflex doesn't really have a code style)